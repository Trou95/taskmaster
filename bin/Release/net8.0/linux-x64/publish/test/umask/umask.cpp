#include <sys/stat.h>

#include <iostream>

int main() {
    int current_umask = umask(0);
    printf("Current Umask: %04o\n", current_umask);
}

