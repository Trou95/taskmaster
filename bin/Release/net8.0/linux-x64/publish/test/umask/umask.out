Current Umask: 0111
